pkg/util/githubutil/githubutil.go

```diff
@@ -76,6 +75,16 @@ func CheckVersionExist(version, repository string) (*github.RepositoryRelease, e
 	return release, err
 }
 
+// GetSHAFromVersion returns sha commit hash against a release
+func GetSHAFromVersion(version, repository string) (string, error) {
+	client := github.NewClient(nil)
```

Comment:
why nil argument.

Reply:
It will create the http client internally

Reply:
Ok can we have that as a default behavior and avoid passing nil unless in some other place you are constructing the client and sending it